_api/hasura/cron/syncCoSouls.ts

```diff
@@ -85,7 +88,14 @@ export async function syncCoSouls() {
       await updateCheckedAt(cosoul.id);
     }
   }
-  const message = `${cosouls.length} CoSouls updated`;
+  let success = true;
+  if (updated.length > 0) {
+    success = await updateCoSoulOnChain(updated);
```

Comment:
can we rename these variables.

using `updated` as the list of cosouls that are gathered prior to be updated is confusing. it seems this should really be named `pending` or `to_be_updated` or `needSync`.

additionally, it seems unconventional to set `success = true` prior to doing the update. I would set success = false, do update, if successful then set to true. or some other pattern that is clearer and less error prone. @sshmm